lang/java/compiler/src/main/java/org/apache/avro/compiler/specific/SpecificCompiler.java

```diff
@@ -929,75 +919,6 @@ public static void main(String[] args) throws Exception {
     compileProtocol(new File(args[0]), new File(args[1]));
   }
 
-  public static final class Slf4jLogChute implements LogChute {
```

Comment:
Removing this class would break the API. Despite it is unlikely that any Avro client used this class (and very unfortunate that it is public, it seems that it isn't used anywhere else apart from this class), would it be possible to keep this, and mark as deprecated?